source/beamline/SGM/SGMHexapod.h

```diff
@@ -3,11 +3,15 @@
 
 #include <QObject>
 #include "beamline/AMControlSet.h"
+
+#define HEXAPOD_POSITION_TOLERANCE 0.008
```

Comment:
Is the position tolerance 0.008 or 0.0008?  Based on our conversations I was under the impression it is the latter.